.ci/jenkins_job_build.sh

```diff
@@ -264,7 +293,7 @@ else
 	echo "WARN: Kata runtime is not installed"
 fi
 
-if [ -n "${METRICS_CI}" ] || [ -n "${METRICS_CI_CLH_BAREMETAL}" ]; then
+if [ -n "${METRICS_CI}" ]; then
```

Comment:
@jcvenegas so we will rename the jobs for metrics to have export METRICS_CI_PROFILE="clh-baremetal"? the ones that run with CLH

Reply:
@GabyCT  we should export the CI_JOB to CLOUD-HYPERVISOR-METRICS-BAREMETAL , the profile is already set on this job id, https://github.com/kata-containers/tests/pull/2653/files#diff-25960c2e9b74d48d51e5a93d8df4455aR282